### PR TITLE
updated github templates

### PR DESCRIPTION
**Pull request by thetif**
These are the new templates that we worked on together. Please take a look and approve or comment on them.

### This pull request changes...

- updates design, issue, and PR templates
- removes dev template
- creates content, feature, and bug templates

### This is how to verify this change...

- for each file, click on the ... and then click on View file to see what the file will look like

### This pull request is ready to merge when...

- [x] Changelog is updated as appropriate
- [ ] Everyone has approved the templates

### CODE REVIEW

**Review by cthulhuplus — APPROVED** [review]

**Review by beparticular — APPROVED** [review]

**Review by akuas — APPROVED** [review]
The templates look good.

**Review by tbolt — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- [ ] Accessibility
+- [ ] Functional
+- [ ] Usability
+- [ ] API
+- [ ] Regression
+
+**Role Affected-** (If not stated, ALL)
+
+**Browser-** (if it only happens in a single browser)
```
Comment (knollfear) [inline .github/ISSUE_TEMPLATE/bug.md:21]: We explicitly only support FF, Chrome and IE, maybe put this below repro steps and include checkboxes for "Duplicated in:" [] FF, [] Chrome, [] IE

**Review by eStar386 — APPROVED** [review]
Thanks, approved. I also agree with Knoll's suggestions.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+**This pull request also touches…**
 
-### This pull request is ready to merge when...
+- possible side effects of this change
 
-- [ ] Develop has updated automated tests (and all tests are passing)
-- [ ] This code has been reviewed by someone other than the original author
-- [ ] QA has manually tested and created bugs for any issues found (beyond minor fixes)
-- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
+**This pull request was tested in the follow ways…**
```
Comment (knollfear) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: These were performed by the developer or the reviewer?  If this is the template I use to open a PR it seems like the developer, but also the final tester probably wants to add to this list.

Comment (thetif) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: I was looking at this as the developer letting the QA know what was already tested. It was a suggestion made by Richie and how his team is currently working. We can try it out and see if we would like to keep it.

**Review by knollfear — APPROVED** [review]

**Pull request merged**